pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -1239,6 +1239,12 @@ func doNetworkInstanceActivate(ctx *zedrouterContext,
 	if status.Type == types.NetworkInstanceTypeSwitch {
 		// switched NI is not probed and does not have a CurrentUplinkIntf
 		status.IfNameList = getIfNameListForLLOrIfname(ctx, status.Logicallabel)
+		if len(status.IfNameList) > 1 {
```

Comment:
does it make sense to check if > 0 also?

Reply:
We'd expect it to be either zero or one; zero if there is no port configured or if the port is not yet available (e.g., assigned to pciback), and one otherwise.